packages/frameworks/src/frameworks.ts

```diff
@@ -1576,11 +1543,47 @@ export const frameworks = [
     getOutputDirName: async () => 'public',
     defaultVersion: '0.13.0',
   },
+  {
+    name: 'Vite',
+    slug: 'vite',
+    //demo: 'https://vite.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/vite.svg',
+    tagline:
+      'Vite is a new breed of frontend build tool that significantly improves the frontend development experience.',
+    description: 'A Vue.js app, created with Vite.',
+    website: 'https://vitejs.dev',
+    envPrefix: 'VITE_',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"vite":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`yarn build` or `npm run build`',
+      },
+      devCommand: {
+        placeholder: '`yarn dev` or `npm run dev`',
+      },
+      outputDirectory: {
+        value: 'dist',
+      },
+    },
+    devCommand: 'yarn dev',
+    buildCommand: 'yarn build',
```

Comment:
This needs to be the actual build command

Reply:
Added based on https://vitejs.dev/guide/#command-line-interface